scripts/codebuild/codebuild.sh

```diff
@@ -57,11 +60,19 @@ yarn install || { echo "yarn install failed" ;exit 1; }
 
 pushd scripts/gatsby-source-iiif/
   yarn install
+
   echo "${magenta}----- Build ENV Config -------${reset}"
-  node setupEnv.js ${PARAM_CONFIG_PATH} > ${ENV_FILE} --unhandled-rejections=strict
-  echo "path="
-  echo ${PARAM_CONFIG_PATH}
-  node setupEnv.js ${PARAM_CONFIG_PATH}
+  # add the keys at /all/static/hostname to the env
+  # node setupEnv.js ${PARAM_CONFIG_PATH} > './ssm-params.txt' --unhandled-rejections=strict
+
+  # put the env sourced vars in the environment file.
+  echo "S3_DEST_BUCKET='${S3_DEST_BUCKET}'" >> $ENV_FILE
+  echo "GRAPHQL_KEY_BASE='${GRAPHQL_KEY_BASE}'" >> $ENV_FILE
+  echo "SEARCH_URL='${SEARCH_URL}'" >> $ENV_FILE
+
+  # add the app sync keys to the env
+  node setupEnv.js ${GRAPHQL_API_KEY_BASE_PATH} >> ${ENV_FILE} --unhandled-rejections=strict
+  source ./ssm-params.txt
```

Comment:
I think this can be removed since since preload-env.sh creates/sources ssm-params.txt